File: Php/codice/L09_Libreria/src/Model/Libro.php
<?php

namespace App\Model;//corrisponde al package name in Java

class Libro {
    //incapsulamento: prop private e metodi pubblici
    private  $id;
    private  $titolo;
    private  $pagine;
    private  $prezzo;
    private  $editore_id;

    public function __get($name){
        return $this->$name;
    }

    public function __set($name, $value){
        $this->$name = $value;
    }

}

File: Java/workspace-fad/Prj15_LibreriaWeb/src/main/java/model/Libro.java
package model;

public class Libro {

	private int id;
    private String titolo;
    private int pagine;
    private double prezzo;
    private int editoreId;
    
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getTitolo() {
		return titolo;
	}
	public void setTitolo(String titolo) {
		this.titolo = titolo;
	}
	public int getPagine() {
		return pagine;
	}
	public void setPagine(int pagine) {
		this.pagine = pagine;
	}
	public double getPrezzo() {
		return prezzo;
	}
	public void setPrezzo(double prezzo) {
		this.prezzo = prezzo;
	}
	public int getEditoreId() {
		return editoreId;
	}
	public void setEditoreId(int editoreId) {
		this.editoreId = editoreId;
	}
    
    
    
}

